radio/src/targets/horus/extmodule_driver.cpp

```diff
@@ -167,7 +167,8 @@ void extmoduleSerialStart()
   EXTMODULE_TIMER->EGR = 1; // Restart
   EXTMODULE_TIMER->CCMR2 = TIM_CCMR2_OC3M_1 | TIM_CCMR2_OC3M_0;
 #else
-  EXTMODULE_TIMER->CCER = TIM_CCER_CC1E | (inverted ? 0 : TIM_CCER_CC1P);
```

Comment:
Please remove these changes, we will figure out what to do in a much broader rewrite of these drivers.